### PR TITLE
 Fix deprecated collections.abc.Mapping import

### PR DESCRIPTION
**Pull request by yohanboniface**
Thanks :)

### CODE REVIEW

File: pymodm/common.py [hunk 1]
```diff
@@ -14,7 +14,11 @@
 
 import re
 
-from collections import Mapping
+try:
+    from collections.abc import Mapping
+except ImportError:  # Python < 3.7
```
[the remark this answers is not in the document]
Comment (yohanboniface) [reply pymodm/common.py]: Done :)

**Comment by yohanboniface** [comment]
> How about we move this import to pymodm/compat.py?

Done :)

File: pymodm/compat.py [hunk 2]
```diff
@@ -15,6 +15,11 @@
 """Tools for Python 2/3 compatibility."""
 import sys
 
+try:
+    import collections.abc as abc
+except ImportError:
+    # Python < 3.3
+    import collections as abc  # noqa
```
Comment (ShaneHarvey) [inline pymodm/compat.py]: What was the warning you needed to disable?

Comment (yohanboniface) [reply pymodm/compat.py]: F401 'collection as abc' imported but not used

**Comment by yohanboniface** [comment]
Anything more needed here? :)

Thanks!

**Comment by ShaneHarvey** [comment]
Thanks!

**Pull request merged**